source/common/amqpClient.js

```diff
@@ -3,335 +3,261 @@
 // SPDX-License-Identifier: Apache-2.0
 
 'use strict';
+var util = require('util');
+var os = require("os");
+var path = require('path');
 var fs = require('fs');
 var amqp = require('amqplib');
-var log = require('./logger')
-  .logger.getLogger('AmqpClient');
+var logger = require('./logger').logger;
+
+var log = logger.getLogger('AmqpClient');
 var cipher = require('./cipher');
-var TIMEOUT = 2000;
-var REMOVAL_TIMEOUT = 7 * 24 * 3600 * 1000;
-var util = require('util');
+var TIMEOUT = 5000;
+var purpose = path.basename(!process.pkg ? __dirname : path.dirname(process.execPath));
+var pid = process.pid;
 
 const RPC_EXC = {
-  name: 'owtRpc',
-  type: 'direct',
-  options: {
-    autoDelete: true,
-    durable: false,
-  },
+    name: 'owtRpc',
+    type: 'direct',
+    options: {
+        autoDelete: false,
+        durable: true,
+    }
 };
 
 const MONITOR_EXC = {
-  name: 'owtMonitoring',
-  type: 'topic',
-  options: {
-    autoDelete: false,
-    durable: false,
-  },
+    name: 'owtMonitoring',
+    type: 'topic',
+    options: {
+        autoDelete: false,
+        durable: true,
+    }
 };
 
 const Q_OPTION = {
-  durable: false,
-  autoDelete: true,
+    durable: true,
```

Comment:
Why set durable to true?

Reply:
> Why set durable to true?

we found if set durable=false, when the first leader down and up later become follower, the new leader will sync itself data to the older leader which had become follower，this action might waste many times. 
set durable=true might let queue data sync quick.it was suggest by our operation & maintenance engineer